packages/lime-ap-watchping/Makefile

```diff
@@ -20,7 +20,7 @@ define Package/$(PKG_NAME)
   TITLE:= Watchping hooks to manage AP SSID if network problems
   MAINTAINER:=Pau Escrich <p4u@dabax.et>
   URL:=http://libremesh.org
-  DEPENDS:= +watchping
+  DEPENDS:= +watchping +lime-system
```

Comment:
I am not sure `lime-system` is really really needed, did you checked the code or consulted with both Paus?

Reply:
The reasoning with references to the code is reported in the PR description.
@aparcar @p4u

Reply:
@G10h4ck do you agree?